doc/_extensions/static/page_filter.js

```diff
@@ -0,0 +1,171 @@
+const all_dropdowns = []
+
+/**
+ * Show all options or hide all except one
+ * @param {String}  option The option to display
+ * @param {Element} dropdown The related dropdown
+ */
+function displayOption(option, dropdown) {
+
+  dropdown.value = option;
+
+  document.querySelectorAll(".hideable").forEach(e => e.hidden = false);
+  var filters = all_dropdowns
+    .map(dp => dp.options[dp.selectedIndex].value)
+    .filter(val => val !== "all");
+
+  if (filters.length == 0) return;
+
+  var negativeFilters = filters
+    .filter(val => val.charAt(0) === "!")
+    .map(val => val.substring(1));
+  filters = filters.filter(val => val.charAt(0) !== "!")
+
+  let selector = "";
```

Comment:
nit: this file should likely make consistent usage of var/let

Reply:
Fixed, thanks!